react-components/src/components/Architecture/SettingsButton.tsx

```diff
@@ -346,3 +347,7 @@ const StyledSlider = styled(Slider)`
   display: flex;
   justify-content: space-around;
 `;
+
+const StyledMenu = styled(Menu)`
+  z-index: 998;
+`;
```

Comment:
And is this Z-index needed (seems to be the same component as the above z-index)